rules/Proj0800.md

```diff
@@ -0,0 +1,52 @@
+# Proj0800: Configure Central Package Version Management explicitly
+Dependency management is a core feature of NuGet. Managing dependencies for a
+single project can be easy. Managing dependencies for multi-project solutions
+can prove to be difficult as they start to scale in size and complexity. In
+situations where you manage common dependencies for many different projects,
+you can leverage NuGet's central package management (CPM) features to do all of
+this from the ease of a single location.
+
+Hence, this rules advises to enable Central Package Version Management via
+enabling it with `<DevelopmentDependency>true</DevelopmentDependency>`.
```

Comment:
You are talking about DevelopmentDependency here, but in the implementation PR it is about ManagePackageVersionsCentrally?

Reply:
Well spotted. Should have been `<ManagePackageVersionsCentrally>`.